scripts/run-mock-api.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -eux
+
+# スクリプトのディレクトリを基準にパスを設定
+SCRIPT_DIR=$(dirname "$0")
+
+# Dockerコンテナをバックグラウンドで起動し、指定された設定ファイルを使用してモックサーバーを立ち上げる
+docker run -d --mount type=bind,source="${SCRIPT_DIR}/../src/api/mock/front-back-api-mock.json",target=/data,readonly -p 4010:4010 mockoon/cli:latest -d /data -p 4010
```

Comment:
このままだと実行ごとにDockerコンテナが増えてしまって手動で消さないといけないので、`--rm`オプションを付けると便利かな、と思います
```suggestion
docker run --rm -d --mount type=bind,source="${SCRIPT_DIR}/../src/api/mock/front-back-api-mock.json",target=/data,readonly -p 4010:4010 mockoon/cli:latest -d /data -p 4010
```